### ISSUE TITLE
Browse topics boxes on front page - re-order

### ISSUE THREAD

**Issue by rossfarrugia** [pharmaverse/pharmaverse#26]
@epijim  Support box should come at the end and any way to show all 5 alongside each other rather than currently only 4 nd then 1 underneath? Shout if this is an update i could make if you point me in right direction

**Comment by epijim** [comment]
@rossfarrugia - this is reactive at the moment, so it tries to match the screen size (e.g. number is max 4, but ranges from 1-4 across depending on screen size). With the content arranged the way it is now inside the boxes, I can't see how we'd fit 5 across (I tried playing with col sizes, but titles get messed up as it's too narrow). 

What if we went the other way? e.g. we make the boxes wider, so on a normal perspective it's 3 on top and 2 underneath but the boxes are more square rather than tall rectangles (again reactive so switching to 2 or 1 across on the fly).

What is looks like now below.

![image](https://user-images.githubusercontent.com/2760096/146177958-69ffd947-daef-4fa0-9c93-034785b933b6.png)
